config/config.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import (
+	"github.com/spf13/pflag"
+
+	"github.com/anchore/fangs/config/log"
+)
+
+type Config struct {
+	Log        log.Log
+	AppName    string `json:"-" yaml:"-" mapstructure:"-"`
+	ConfigFile string `json:"config,omitempty" yaml:"config,omitempty" mapstructure:"-"`
+}
+
+func NewConfig(appName string) Config {
+	return Config{
+		Log:     log.NewDiscard(),
+		AppName: appName,
+	}
+}
+
+func (r *Config) AddFlags(flags *pflag.FlagSet) {
+	flags.StringVarP(&r.ConfigFile, "config", "c", r.ConfigFile, "configuration file")
```

Comment:
question: should long name of `config` and short name of `c` be options on the config?

Reply:
I think if a user wants a different binding, they should just not call this function and make their own. The idea here is to make things consistent

Reply:
ahh point taken 👍